Assets/Scripts/LobbyManager.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class LobbyManager : MonoBehaviour
+{
+    public void Level1() {
```

Comment:
You have created multiple methods which are doing basically the same thing. A better way is to create a generic method to handle these type cases. Create a LoadLevel method and pass an integer as a parameter which would be your level index. Always avoid copy pasting code.